spec/system/candidate_interface/candidate_entering_work_history_spec.rb

```diff
@@ -132,4 +137,20 @@ def and_i_change_the_job_title
   def then_i_should_see_my_updated_job
     expect(page).to have_content('Chief Executive Officer')
   end
+
+  def when_i_mark_this_section_as_completed
+    check t('application_form.work_history.review.i_have_completed')
+  end
+
+  def and_i_click_on_continue
+    click_button t('application_form.work_history.review.button')
+  end
+
+  def then_i_should_see_the_form
+    expect(page).to have_content(t('page_titles.personal_details'))
```

Comment:
Don't think this is what we should expect.

Reply:
Oh yeah!